react-native-aztec/ios/RNTAztecView/RCTAztecView.swift

```diff
@@ -571,6 +571,9 @@ class RCTAztecView: Aztec.TextView {
 extension RCTAztecView: UITextViewDelegate {
 
     func textViewDidChangeSelection(_ textView: UITextView) {
+        guard isFirstResponder else {
```

Comment:
Maybe we could add this statement to the previously existing `guard`?
Not sure if we get an advantage by having them separated.